### PR TITLE
315 Shrink bottom margin on labels

### PR DESCRIPTION
**Pull request by ericsorenson**
image

### CODE REVIEW

File: web-client/src/views/LogIn.jsx [hunk 1]
```diff
@@ -4,22 +4,21 @@ import React from 'react';
 
 import ErrorNotification from './ErrorNotification';
 
-export default connect(
```
Comment (wbyoko) [inline web-client/src/views/LogIn.jsx:7]: where is LogIn being used?

Comment (ericsorenson) [reply web-client/src/views/LogIn.jsx:7]: it's the mock log in screen that we only use for local development

Comment (wbyoko) [reply web-client/src/views/LogIn.jsx:7]: ok, your switching to a named export but you are not destructuring it anywhere in this PR. Not sure if that breaks anything.

Comment (kkoskelin) [reply web-client/src/views/LogIn.jsx]: update required in `App.jsx` I believe

**Review by wbyoko — APPROVED** [review]

**Pull request merged**